docs/developer/editors.md

```diff
@@ -0,0 +1,175 @@
+# Editors
+
+Atomist is about automating more of our own work, and that includes the work of writing code. Not all code, of course! But the repetitive parts of coding, or the code that you want to change across many projects.
+
+If the change would be so much tedious work to do by hand that you wouldn't even bother, consider automating it.
+
+Atomist has some key tools that make automating code feasible: tools for meaningful code manipulation functions, and [command handlers](commands.md) to turn your functions into pull requests on one or more projects.
```

Comment:
"Atomist provides tools that make..."

Reply:
I'm not sure what "tools for meaningful code manipulation functions" means. Do you mean "tools for parsing and manipulating code" or "tools for creating code manipulation functions"?

Reply:
I think we should use "command automation", like you use below, rather than "command handler".